.travis.yml

```diff
@@ -0,0 +1,6 @@
+dist: trusty
+sudo: false
```

Comment:
`sudo` is `false` by default, you can remove this line.
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments